v3/src/components/web-view/web-view.tsx

```diff
@@ -26,6 +30,68 @@ export const WebViewComponent = observer(function WebViewComponent({ tile }: ITi
       <div className="codap-web-view-iframe-wrapper">
         <iframe className="codap-web-view-iframe" ref={iframeRef} src={webViewModel.url} />
       </div>
+      <WebViewDropOverlay />
```

Comment:
Any issues with rendering this over the plugin potentially affecting the behavior of the plugin? A priori, I might have thought that the overlay would only be rendered during a drag, which would mitigate any potential impact on the plugin at other times.

Reply:
Unless an attribute is being dragged, `WebViewDropOverlay` returns `null` 👍

Reply:
🤦 Right, you are. 😉